torchgeo/datasets/ucmerced.py

```diff
@@ -143,6 +144,21 @@ def __init__(
             is_valid_file=is_in_split,
         )
 
+    def _load_image(self, index: int) -> Tuple[Tensor, Tensor]:
+        """Load a single image and it's class label.
+
+        Args:
+            index: index to return
+        Returns:
+            the image
+            the image class label
+
+        .. versionadded:: 0.4.2
```

Comment:
We don't need to document this, it isn't public and technically doesn't change the API since the base class also had this method.